bower.json

```diff
@@ -26,7 +26,8 @@
     "purescript-numbers": "^5.0.0",
     "purescript-oidc-crypt-utils": "^7.0.1",
     "purescript-pathy": "^4.0.0",
-    "purescript-uri": "^4.0.0"
+    "purescript-uri": "^4.0.0",
+    "purescript-sqlsquare": "purescript-sql-squared#^0.8.2"
```

Comment:
`purescript-sql-squared: ^0.8.2`?

Reply:
it's should be related to `https://github.com/slamdata/purescript-sql-squared/issues/32`